docs/admin/logical-replication.rst

```diff
@@ -50,6 +50,15 @@ A publication is created using the
 altered or dropped using corresponding commands. The individual tables can be
 added and removed dynamically using ALTER PUBLICATION.
 
+.. CAUTION::
+
+    Publishing cluster must have
+    :ref:`setting <sql-create-table-soft-deletes-enabled>`
+    set to ``true`` so that subscribing cluster can catch up all changes made
+    during replication pause caused by network issues or explicitly done by a user.
+    Also, :ref:`setting <sql-create-table-soft-deletes-retention-lease-period>`
+    should not be greater than ``replication.logical.reads_poll_duration``.
```

Comment:
```suggestion
    should be greater than ``replication.logical.reads_poll_duration``.
```

Reply:
or equal?

Reply:
added equal in https://github.com/crate/crate/pull/12138/commits/4eb76d902581bf10da6bc0fcb6b05f8d77065b3f